tests/txt.test.js

```diff
@@ -125,6 +126,53 @@ describe('validateCodeComments', () => {
   })
 })
 
+describe('validateIDIndicator', () => {
+  test('should return no warnings in SUBMISSION mode', async () => {
+    const doc = cloneDeep(baseTXTDoc)
+    const result = await validateIDIndicator(doc, { mode: MODES.SUBMISSION })
+    expect(result).toEqual([
+      new ValidationError(
+        'ID_INDICATOR_MISSING',
+        'Document does not contain an ID indication.',
+        { ref: 'https://authors.ietf.org/en/drafting-in-plaintext#checklist' }
+      )
+    ])
+  })
```

Comment:
The description and the test do not align. The test is correct, the description is not.